src/main/common/filter.c

```diff
@@ -68,15 +79,26 @@ FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
 
 // PT2 Low Pass filter
 
+// PTn cutoff correction = 1 / sqrt(2^(1/n) - 1)
+#define CUTOFF_CORRECTION_PT2 1.553773974f
+
 FAST_CODE float pt2FilterGain(float f_cut, float dT)
 {
-    // PTn cutoff correction = 1 / sqrt(2^(1/n) - 1)
-    #define CUTOFF_CORRECTION_PT2 1.553773974f
-
     // shift f_cut to satisfy -3dB cutoff condition
     return pt1FilterGain(f_cut * CUTOFF_CORRECTION_PT2, dT);
 }
 
+// Calculates filter gain based on delay (time constant of filter) - time it takes for filter response to reach 63.2% of a step input.
+float pt2FilterGainFromDelay(float delay, float dT)
+{
+    if (delay == 0) {
+        return 1.0f; // gain = 1 means no filtering
+    }
+
+    const float cutoffHz = 1.0f / (M_PIf * delay * CUTOFF_CORRECTION_PT2);
+    return pt2FilterGain(cutoffHz, dT);
```

Comment:
`CUTOFF_CORRECTION_PT2` is getting cancelled out in `pt2FilterGain()`. Is this intentional? Because then we can simplify this and we no longer need the cutoff correction.

Reply:
Yes, it does cancel. But written this way, it is obvious that cutoff frequency is altered for 2nd order PT (and you can reason about cutoff frequency used), to get desired delay. IMO it is slightly better this way.